tapchannel/aux_funding_controller.go

```diff
@@ -1264,7 +1264,7 @@ func (f *FundingController) processFundingMsg(ctx context.Context,
 		)
 		if err != nil {
 			return tempPID, fmt.Errorf("unable to create "+
-				"commitment: %v", err)
+				"commitment: %w", err)
```

Comment:
Weird that the linter didn't catch this, thought I had addressed all these areas.

Reply:
Yeah, I also thought we turned on that linter...